operations/compute_rolling_upgrades/Upgrade_Compute_Nodes_with_CRUS.md

```diff
@@ -1,28 +1,30 @@
 # Upgrade Compute Nodes with CRUS
 
-Upgrade a set of compute nodes with the Compute Rolling Upgrade Service \(CRUS\). Manage the workload management status of nodes and quiesce each node before taking the node out of service and upgrading it. Then reboot it into the upgraded state and return it to service within the workload manager \(WLM\).
+Upgrade a set of compute nodes with the Compute Rolling Upgrade Service \(CRUS\). Manage the workload management status of nodes and quiesce each node before taking the node
+out of service and upgrading it. Then reboot it into the upgraded state and return it to service within the workload manager \(WLM\).
 
-### Prerequisites
+**Note:** CRUS is deprecated and will be removed in an upcoming CSM release.
+
+## Prerequisites
 
 - The Cray command line interface \(CLI\) tool is initialized and configured on the system.
 - A Boot Orchestration Service \(BOS\) session template describing the desired states of the nodes being upgraded must exist.
 
-### Procedure
+## Procedure
 
-1.  Create and populate the starting node group.
+1. Create and populate the starting node group.
 
-    This will be the group of nodes that will be upgraded.
+    This is the group of nodes that will be upgraded.
 
-    1.  Create a starting node group \(starting label\).
+    1. Create a starting node group \(starting label\).
 
-        Label names are defined by the user and the names used in this procedure are only examples. The label name used in this example is slurm-nodes.
+        Label names are defined by the user. The names used in this procedure are only examples. The label name used in this example is `slurm-nodes`.
 
         ```bash
-        ncn# cray hsm groups create --label slurm-nodes \
-        --description 'Starting Node Group for my Compute Node upgrade'
+        ncn# cray hsm groups create --label slurm-nodes --description 'Starting Node Group for my Compute Node upgrade'
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815